packages/terra-notification-dialog/src/NotificationDialog.scss

```diff
@@ -0,0 +1,106 @@
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+
+    @media screen and (max-width: 768px) {
+      width: calc(100% - 2 * var(--notification-dialog-width, 10px));
+    }
+  }
+
+  .notification-dialog-body {
+    background-color: var(--notification-dialog-body-color, #fff);
+    display: inline-flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    height: 100%;
+    overflow: auto;
+    padding: var(--notification-dialog-body-padding, 1.25rem);
+    width: 100%;
+  }
+
+  .notification-dialog-body-text {
+    background-color: var(--notification-dialog-body-color, #fff);
+    font-size: var(--notification-dialog-body-font-size, 1.25rem);
+  }
+
+  // This flex container is needed for IE 10 / IE 11 to render correctly
+  .notification-dialog-inner-wrapper {
+    display: flex;
+    flex-direction: row;
+    max-height: 80vh;
+  }
+
+  .notification-dialog-container {
+    display: flex;
+    flex-direction: column;
+    max-height: inherit; // `inherit` value needed for IE10 to scroll modal-body when overflowed
+    width: 100%; // Needed for IE to wrap content correctly in flexbox
+  }
+
+  .header-body {
+    background-color: var(--notification-dialog-header-color, #f9f9f9);
+    border: var(--notification-dialog-header-border, 0.0625rem solid #dedfe0);
+    font-size: var(--notification-dialog-header-font-size, 1.125rem);
+    padding: var(--notification-dialog-body-padding, 0.8rem);
+    text-align: left;
+  }
+
+  .footer-body {
+    background-color: var(--notification-dialog-footer-color, #f9f9f9);
+    border: var(--notification-dialog-footer-border, 0.0625rem solid #dedfe0);
+    flex: 0 0 auto;
+    font-size: var(--notification-dialog-footer-font-size, 0.8rem);
+  }
+
+  .icon-div {
+    > svg {
+      height: 2rem;
+      margin-right: var(--terra-notification-dialog-icon-margin-right, 1.25rem);
+      width: 2rem;
+    }
+  }
+
+  .title {
+    color: var(--terra-notification-dialog-title-color, #000);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-size: var(--terra-notification-dialog-title-font-size, 1rem);
+    font-weight: var(--terra-notification-dialog-title-font-weight, bold);
+    text-align: var(--terra-notification-dialog-title-text-align, left);
+    width: 100%;
+    word-wrap: break-word;
+  }
+
+  .message {
+    color: var(--terra-notification-dialog-message-color, #000);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-size: var(--terra-notification-dialog-message-font-size, 1rem);
+    padding-top: var(--terra-notification-dialog-message-padding-top, 1rem);
+    text-align: var(--terra-notification-dialog-message-text-align, left);
+    width: 100%;
+    word-wrap: break-word;
+  }
+
+  .actions {
+    display: flex;
+    flex: 0 0 auto;
+    flex-direction: var(--terra-notification-dialog-actions-flex-direction-small-width, row);
+    justify-content: flex-end;
+    padding: var(--terra-notification-dialog-actions-padding, 0.4rem);
```

Comment:
Let's update this to  `padding: var(--terra-notification-dialog-actions-padding, 0.28571rem 0.35714rem);`
and move it to the `.footer-body` so it better matches with `.header-body` also having padding.